civihr_employee_portal/views/templates/views-view-table--Documents--block.tpl.php

```diff
@@ -224,9 +224,18 @@ function filterTable(statusId) {
 
     // Listen for ready event when T&A finishes loading all modules
     document.addEventListener('taReady', function () {
+      var selectedFilter = window.location.hash.substring(1);
```

Comment:
Angular bootstrap removes the window hash and query string, so had to do it before that happened.